tests/MaintainersTest.php

```diff
@@ -8,9 +8,201 @@
 
 namespace Keboola\ManageApiTest;
 
+use Keboola\ManageApi\Client;
+use Keboola\ManageApi\ClientException;
+
 class MaintainersTest extends ClientTestCase
 {
+    private $normalUser;
+    
+    private $superAdmin;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->normalUser = $this->normalUserClient->verifyToken()['user'];
+        $this->superAdmin = $this->client->verifyToken()['user'];
+    }
+
+    /**
+     * Clean up any created maintainers or users added
+     */
+    public function tearDown()
```

Comment:
please move this to `setUp`. It is safer because in tear down in should be not executed on some fatal error or die.